### ISSUE TITLE
portal.azure.com / api: breakage

### ISSUE THREAD

**Issue by Donran** [uBlockOrigin/uAssets#15042]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625) (also mind browser built-in blockers)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://portal.azure.com/`
`api.applicationinsights.azure.com`

### Category

breakage

### Describe the issue

uBlock blocks real requests for the log query api in the Azure portal. You can see in the network tab that these requests are "Blocked by uBlock Origin"

My specific case is a POST towards the `https://api.applicationinsights.azure.com/v1/apps/{GUID}/query` endpoint.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![Screenshot 2022-09-26 110053](https://user-images.githubusercontent.com/4838348/192236515-26bf16b8-c061-4e49-9683-61971383b655.png)
I can't post a full screenshot of the website as it might contain sensitive information. But you can see the "Error retrieving data" message here. And I can see the blocked requests.

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.44.4
Firefox: 105
filterset (summary): 
  network: 85785
  cosmetic: 41425
  scriptlet: 18209
  html: 768
listset (total-discarded, last updated): 
  added: 
    SWE-1: 4272-2, 3d.45m
  default: 
    user-filters: 0-0, never
    ublock-filters: 33585-39, 3d.42m
    ublock-badware: 4209-0, 3d.44m
    ublock-privacy: 266-0, 13d.23m
    ublock-abuse: 76-0, 13d.22m
    ublock-unbreak: 1873-0, 3d.43m
    easylist: 62462-616, 1h.32m
    easyprivacy: 30639-1709, 1h.30m
    urlhaus-1: 7407-0, 1h.29m
    plowe-0: 3674-3, 13d.24m
    ublock-quick-fixes: 252-0, 1h.28m
filterset (user): [empty]
trustedset: 
  added: [array of 4 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 1487 ms (selfie)
  maxAssetCacheWait: 328 ms
popupPanel: 
  blocked: 13
  blockedDetails: 
    azure.com: 8
    microsoft.com: 4
    msecnd.net: 1
```
</details>

**Comment by Donran** [comment]
> > uBlock blocks real requests for the log query api in the Azure portal
> 
> > My specific case is a POST towards the `https://api.applicationinsights.azure.com/v1/apps/{GUID}/query` endpoint.
> 
> 1- Open the uBO's logger: https://github.com/gorhill/uBlock/wiki/The-logger 2- Repeat the steps to get the issue. 3- Go back into the uBO's logger and then copy and paste here all blocked requests.

```
+1
||applicationinsights.azure.com^
--
portal.azure.com
1
xhr
https://api.applicationinsights.azure.com/v1/apps/fake-guid/query?timespan=2022-09-25T09:12:15.280Z/2022-09-26T09:12:15.280Z
```
```
+1
||applicationinsights.azure.com^
--
portal.azure.com
1
xhr
https://api.applicationinsights.azure.com/v1/apps/fake-guid/query?timespan=2022-09-25T09:12:15.280Z/2022-09-26T09:12:15.280Z
```
</details>
These requests are being blocked, with a fake guid.

**Comment by evancjohnson** [comment]
@MasterKia @mapx- I am new so I apoligize if I am tagging the wrong folks and I would be happy to create a new issue if needed. 

I am also experiencing this issue and I think it has been miscategorized, to my understanding `applicationinsights.azure.com` does not perform any tracking and only provides capabilities needed for the azure portal. I believe data is ingested at `dc.services.visualstudio.com/v2` and then enriched by azure services once it hits their network. When viewing these analytics, the Azure portal calls `api.applicationinsights.azure.com` to query for data to display in the front end. As far as I know all MSFT collected data goes through vortex and all customer collected insghts go through `dc.services.visualstudio.com' 

For the time being I have turned off the extension for portal.azure.com and the functionality has been restored.

**Comment by evancjohnson** [comment]
> `||applicationinsights.azure.com^$3p` allows requests initiated by `azure.com`.
> 
> You should force update all your filter lists and test again. https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches

That worked for me, thank you!

**Comment by TomasMalecek** [comment]
`||applicationinsights.azure.com^$3p` causes me issues now in Azure Portal. When looking at failures in an App Insights resource, the initial load is OK, but when I select a different time period, I see errors everywhere and the requests are blocked. They seem to be initiated from a subdomain of azure.net, specifically appinsights.hosting.portal.azure.net.

I create a custom rule that solved the issue for me:
```
@@||api.applicationinsights.azure.com^$domain=portal.azure.net
```

**Comment by TomasMalecek** [comment]
I unchecked "My filters" in the "Filter lists" tab of uBO settings and reproduced the settings while the logger was open. Hope this helps:

<details><summary>Logger output</summary>

| | | | | | | |
|:--- |:--- |:--- |:--- |:--- |:--- |:--- |
| +0 |  |  | portal.azure.com | 1 | xhr | `https://management.azure.com/batch?api-version=2020-06-01` |
| +0 | \|\|applicationinsights.azure.com^$3p | -- | appinsights.hosting.portal.azure.net | 1,3 | xhr | `https://api.applicationinsights.azure.com/v1/apps/3c40ce8b-9a31-48bb-8d15-200ed7e8014e/query` |
| +0 | \|\|applicationinsights.azure.com^$3p | -- | appinsights.hosting.portal.azure.net | 1,3 | xhr | `https://api.applicationinsights.azure.com/v1/apps/3c40ce8b-9a31-48bb-8d15-200ed7e8014e/query` |
| +0 | \|\|applicationinsights.azure.com^$3p | -- | appinsights.hosting.portal.azure.net | 1,3 | xhr | `https://api.applicationinsights.azure.com/v1/apps/3c40ce8b-9a31-48bb-8d15-200ed7e8014e/query` |
| +0 | \|\|applicationinsights.azure.com^$3p | -- | appinsights.hosting.portal.azure.net | 1,3 | xhr | `https://api.applicationinsights.azure.com/v1/apps/3c40ce8b-9a31-48bb-8d15-200ed7e8014e/query` |
| +0 | \|\|applicationinsights.azure.com^$3p | -- | appinsights.hosting.portal.azure.net | 1,3 | xhr | `https://api.applicationinsights.azure.com/v1/apps/3c40ce8b-9a31-48bb-8d15-200ed7e8014e/query` |
| +0 | \|\|applicationinsights.azure.com^$3p | -- | appinsights.hosting.portal.azure.net | 1,3 | xhr | `https://api.applicationinsights.azure.com/v1/apps/3c40ce8b-9a31-48bb-8d15-200ed7e8014e/query` |
| +0 | \|\|applicationinsights.azure.com^$3p | -- | appinsights.hosting.portal.azure.net | 1,3 | xhr | `https://api.applicationinsights.azure.com/v1/apps/3c40ce8b-9a31-48bb-8d15-200ed7e8014e/query` |
| +0 | \|\|applicationinsights.azure.com^$3p | -- | appinsights.hosting.portal.azure.net | 1,3 | xhr | `https://api.applicationinsights.azure.com/v1/apps/3c40ce8b-9a31-48bb-8d15-200ed7e8014e/metrics` |
| +0 | \|\|applicationinsights.azure.com^$3p | -- | appinsights.hosting.portal.azure.net | 1,3 | xhr | `https://api.applicationinsights.azure.com/v1/apps/3c40ce8b-9a31-48bb-8d15-200ed7e8014e/metrics/metadata` |
</details>

**Comment by TomasMalecek** [comment]
Works for me now, @MasterKia, @ryanbr. Thanks!

**Comment by Yuki2718** [comment]
Maybe the blocking rule should be removed. I have azure access too and they constantly change and add domains interconnected very complecatedly.
